charts/kubeai/values.yaml

```diff
@@ -107,6 +107,9 @@ modelAutoscaling:
   # Time window the autoscaling algorithm will consider when calculating
   # the desired number of replicas.
   timeWindow: 10m
+  # The name of the ConfigMap that stores the state of the autoscaler.
+  # Defaults to "{fullname}-autoscaler-state".
+  # stateConfigMapName: "kubeai-autoscaler-state"
```

Comment:
```suggestion
  stateConfigMapName: ""
```

Reply:
this is what we do for other names like secrets.huggingface.name

Reply:
Added